### PR TITLE
refactor: removed `basic` example from tui-editor

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo/src/modules/components/editor-new/editor-new.component.ts [hunk 1]
```diff
@@ -34,73 +34,67 @@ import {ABSTRACT_PROPS_ACCESSOR} from '../abstract/inherited-documentation/abstr
     ],
 })
 export class ExampleEditorNewComponent extends AbstractExampleTuiControl {
-    readonly exampleModule = import(`!!raw-loader!./examples/import/import-module.md`);
-    readonly exampleHtml = import(`!!raw-loader!./examples/import/insert-template.md`);
+    readonly exampleModule = import(`!!raw-loader!.//examples/import/import-module.md`);
+    readonly exampleHtml = import(`!!raw-loader!.//examples/import/insert-template.md`);
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/editor-new/editor-new.component.ts:38]: ?

File: projects/demo-integrations/cypress/support/editor/helpers.ts [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+} from '../shared.entities';
+import {HTML_EDITOR_BASIC_EXAMPLE} from './html';
+
+export function tuiVisitEditorApiPage({
+    content,
+    maxHeight,
+    enableNightMode,
+}: Partial<{content: string; maxHeight: number; enableNightMode: boolean}> = {}): void {
+    cy.viewport(1650, 900).tuiVisit(
+        `${EDITOR_PAGE_URL}/API?tuiMode=null&ngModel=${
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/support/editor/helpers.ts:14]:
```suggestion
        `${EDITOR_PAGE_URL}/API?ngModel=${
```

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/editor-new/editor-new.component.ts (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/editor-new/editor-new.component.ts]: fixed

File: projects/demo-integrations/cypress/support/editor/helpers.ts (hunk 2 above)
Comment (splincode) [reply projects/demo-integrations/cypress/support/editor/helpers.ts]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**